src/main/java/apoc/uuid/UUIDLifeCycle.java

```diff
@@ -0,0 +1,78 @@
+package apoc.uuid;
+
+import apoc.ApocConfiguration;
+import apoc.util.Util;
+import org.neo4j.graphdb.QueryStatistics;
+import org.neo4j.graphdb.Result;
+import org.neo4j.scheduler.JobScheduler;
+import org.neo4j.kernel.internal.GraphDatabaseAPI;
+import org.neo4j.logging.Log;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * @author ab-larus
+ * @since 05.09.18
+ */
+
+public class UUIDLifeCycle {
+
+    public static final int DEFAULT_SCHEDULE = 60;
+    public static JobScheduler.Group UUID_GROUP = new JobScheduler.Group("UUID");
+    private final JobScheduler scheduler;
+    private final GraphDatabaseAPI db;
+    private JobScheduler.JobHandle uuidJobHandle;
+    private Log log;
+
+    public UUIDLifeCycle(JobScheduler scheduler, GraphDatabaseAPI db, Log log) {
+        this.scheduler = scheduler;
+        this.log = log;
+        this.db = db;
+    }
+
+    public void start() {
+        boolean enabled = Util.toBoolean(ApocConfiguration.get("uuid.enabled", null));
+        if (!enabled) return;
+
+        long uuidSchedule = Util.toLong(ApocConfiguration.get("uuid.schedule", DEFAULT_SCHEDULE));
+
+        String label = ApocConfiguration.get("uuid.labels", "");
+        List<String> labels = Arrays.asList(label.split(","));
+
+        System.out.println("labels = " + labels.size());
```

Comment:
Togli tutti i sysout e usa il logger se è necessiario, nella classe hai accesso al logger con `this.log`
Usa per il debug dei parametri `this.log.debug` quindi tutti i sysout passali a quest'ultimo